pkg/security/common/containerutils/utils.go

```diff
@@ -7,13 +7,14 @@
 package containerutils
 
 import (
-	"crypto/sha256"
-	"fmt"
 	"regexp"
 )
 
-// ContainerIDPatternStr is the pattern of a container ID
-var ContainerIDPatternStr = fmt.Sprintf(`([[:xdigit:]]{%v})`, sha256.Size*2)
+// ContainerIDPatternStr defines the regexp used to match container IDs
+// ([0-9a-fA-F]{64}) is standard container id used pretty much everywhere
+// ([0-9a-fA-F]{32}-[0-9]{10}) is container id used by AWS ECS
+// ([0-9a-fA-F]{8}(-[0-9a-fA-F]{4}){4}) is container id used by Garden
+var ContainerIDPatternStr = "([0-9a-fA-F]{64})|([0-9a-fA-F]{32}-[0-9]{10})|([0-9a-fA-F]{8}(-[0-9a-fA-F]{4}){4})"
```

Comment:
You didn't use the one from `util/cgroups` because of the size ?

Reply:
Exactly !